src/ol/proj/index.js

```diff
@@ -739,6 +725,27 @@ ol.proj.getTransformFromProjections = function(sourceProjection, destinationProj
   var transform;
   if (sourceCode in transforms && destinationCode in transforms[sourceCode]) {
     transform = transforms[sourceCode][destinationCode];
+  } else if (ol.ENABLE_PROJ4JS) {
+    var proj4js = ol.proj.proj4_ || window['proj4'];
+    if (typeof proj4js == 'function') {
+      var sourceDef = proj4js.defs(sourceCode);
+      var destinationDef = proj4js.defs(destinationCode);
+
+      if (sourceDef !== undefined && destinationDef !== undefined) {
+        if (sourceDef === destinationDef) {
+          ol.proj.addEquivalentProjections([destinationProjection, sourceProjection]);
+        } else {
+          var proj4Transform = proj4js(destinationCode, sourceCode);
+          ol.proj.addCoordinateTransforms(destinationProjection, sourceProjection,
+              proj4Transform.forward, proj4Transform.inverse);
+        }
+        if (sourceCode in transforms && destinationCode in transforms[sourceCode]) {
+          transform = transforms[sourceCode][destinationCode];
+        }
+      } else {
+        transform = undefined;
```

Comment:
If I don't do that here, the builder complains, but I can't figure out why. I'd be interested if someone has an logical explanation.